Sources/Baggage/BaggageContext.swift

```diff
@@ -64,10 +64,34 @@ public struct BaggageContext {
 
 extension BaggageContext: CustomStringConvertible {
     public var description: String {
-        "\(Self.self)(keys: \(self._storage.map(\.key.name)))"
+        "\(Self.self)(keys: \(self._storage.map { $0.key.name }))"
```

Comment:
Don’t you like KeyPaths? 😁

Reply:
Only because we want to be compatible with old swift versions which didn't have that yet.
Otherwise I love that new syntax :)

Reply:
Do we want that? I think we talked about the min Swift version in `Package.swift` a while ago and decided to set it to 5.2.

Reply:
Right, I though tho that that syntax was 5.3?

Reply:
Nope, that one already came in with 5.2 🥳 https://github.com/apple/swift-evolution/blob/master/proposals/0249-key-path-literal-function-expressions.md

Reply:
Oh awesome :) thanks!